ts/Data/DataTableCore.ts

```diff
@@ -93,7 +93,11 @@ class DataTableCore {
          * @type {string}
          */
         this.id = (options.id || uniqueKey());
+
         this.modified = this;
+        // Remove the modified property to prevent the circular reference.
+        delete this.modified.modified;
```

Comment:
It's not what I would expect from here. Why do we assign `this` to `this.modified` at all in the constructor? And why we need to remove this.modified from here?